Description: Fixed blur icon in the D-Bus notifications in Ubuntu Unity
 This patch was backported from version 2.2.7 which is not yet released.
Author: Pavel Vatagin <pavelvat@gmail.com>
Last-Update: 2012-04-10

--- a/eiskaltdcpp-qt/src/Notification.h
+++ b/eiskaltdcpp-qt/src/Notification.h
@@ -44,7 +44,7 @@ public:
         QVariantList args;
         args << QString("EiskaltDC++");
         args << QVariant(QVariant::UInt);
-        args << QVariant(WulforUtil::getInstance()->getIconsPath() + "/" + "icon_appl.png");
+        args << QVariant(WulforUtil::getInstance()->getIconsPath() + "/" + "icon_appl_big.png");
         args << QString(title);
         args << QString(msg);
         args << QStringList();
